scripted-tests/run/java-io-file-1/tests/CanonicalPathTest.scala

```diff
@@ -1,7 +1,8 @@
 object CanonicalPathTest {
   import Files._
+  import scala.scalanative.runtime.Platform
 
-  def main(args: Array[String]): Unit = {
+  def main(args: Array[String]): Unit = if (!Platform.isWindows) {
```

Comment:
JVM doesn't support these file operations and crashes, but scala-native has support of it on windows and all unit-tests passed (not scripted one). I spoke with Martin about it, we haven't clear decision what to do with this problem.